src/main/java/com/vesoft/nebula/jdbc/statement/NebulaStatementImpl.java

```diff
@@ -204,8 +204,8 @@ public void cancel() throws SQLException {
     }
 
     @Override
-    public SQLWarning getWarnings() throws SQLException {
-        throw ExceptionBuilder.buildUnsupportedOperationException();
+    public SQLWarning getWarnings() {
+        return null;
```

Comment:
could you explain why always return `null` here?

Reply:
This issue answers the question very well #54.
Clickhouse jdbc client is also written in this way and returns null

https://github.com/ClickHouse/clickhouse-java/blob/main/clickhouse-jdbc/src/main/java/com/clickhouse/jdbc/internal/ClickHouseStatementImpl.java
line 628

Reply:
Directly returning `null` may not be the best approach. It would be better if we could return `SQLWarning`. I'd like to merge this PR first. How do you think, @wey-gu?

Thank you for your contribution.